nrf_dm: Upgrading to 0.7.5 version of nrf_dm (#872)
nrfconnect/sdk-nrfxlib: nrf_dm/doc/CHANGELOG.rst

main
****

> 

Reviewer: ```suggestion

All the notable changes added to the |NCS| main are documented in this section.

```

Author: Fixed

Changes

> =======

Reviewer: ```suggestion
=======

```

Author: Fixed

> * Improved the resilience to interference during synchronization.

Reviewer: The resilience of what?

Author: Trying to say that the synchronization procedure will fail less often due to interference

Author: Reworded this a bit

Author: > Increased the synchronization procedure's resilience to interference.

Thanks. Updated to this now.

* Improved the quality of access addresses used internally in the library.

> * ``access_address`` was renamed to ``rng_seed`` in :c:type:`nrf_dm_config_t`.

Reviewer: ```suggestion
* Renamed ``access_address`` to ``rng_seed`` in :c:type:`nrf_dm_config_t`.
```
Makes us sound more active, instead of something just happening.

* :c:func:`nrf_dm_high_precision_calc` now handles out-of-memory errors by returning `NaN`.

nRF Connect SDK v2.1.0
**********************

> All the notable changes added to the |NCS| main are documented in this section.

Reviewer: ```suggestion
All the notable changes added to the |NCS| v2.1.0 release are documented in this section.
```

Author: Fixed
